Content.Shared/Chemistry/Solution/Components/SolutionContainerManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Robust.Shared.Analyzers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.Chemistry.Solution.Components
+{
+    [NetworkedComponent()]
+    [RegisterComponent]
+    [DataDefinition]
+    [Friend(typeof(SolutionContainerSystem))]
+    public class SolutionContainerManager : Component
+    {
+        public override string Name => "SolutionContainer";
+
+        [ViewVariables] [DataField("solutions")]
+        public readonly Dictionary<string, Solution> Solutions = new();
+    }
```

Comment:
That's one sexy class.

```suggestion
    [NetworkedComponent()]
    [RegisterComponent]
    [DataDefinition]
    [Friend(typeof(SolutionContainerSystem))]
    public class SolutionContainerManagerComponent : Component
    {
        public override string Name => "SolutionContainerManager";

        [ViewVariables] [DataField("solutions")]
        public readonly Dictionary<string, Solution> Solutions = new();
    }
```

Reply:
I miss Discord emoji. Especially :agonylimitless: 

It's appropriate.